cloud/google/gce.py

```diff
@@ -58,6 +58,13 @@
     required: false
     default: null
     aliases: []
+  service_account_permissions:
+    version_added: 1.5.1
+    description:
+      - service account permissions (see U(https://cloud.google.com/sdk/gcloud/reference/compute/instances/create), --scopes section for detailed information)
+    required: false
+    default: null
+    aliases: []
```

Comment:
You may want to add the possible choices here.

Ala 

    aliases: []
    choices: ["bigquery", "cloud-platform", "compute-ro", "compute-rw", "computeaccounts-ro", "computeaccounts-rw", "datastore", "logging-write", "monitoring", "sql", "sql-admin", "storage-full", "storage-ro", "storage-rw", "taskqueue", "userinfo-email"]